test/__setup__.ts

```diff
@@ -1,7 +1,7 @@
-import { afterAll, beforeAll, beforeEach, vi } from "vitest";
-import { createMockFactory, MockFactory } from "@prismicio/mock";
-import { setupServer, SetupServerApi } from "msw/node";
+import { MockFactory, createMockFactory } from "@prismicio/mock";
 import AbortController from "abort-controller";
+import { SetupServerApi, setupServer } from "msw/node";
+import { afterAll, beforeAll, beforeEach, vi } from "vitest";
```

Comment:
I don't know how you feel about it, but I tend to prefer test libraries to be listed at the very top, wdyt?

Reply:
Maybe it could be that any `devDependency` is its own group at the very top? We would need to change the Prettier config to a JS file so we can programmatically generate that list.

I don't have a strong opinion on sorting test deps to the top. If we go the `devDependencies` route, `node-fetch` might be added to the top if it's only used in tests, but not if it is used in the library.

Reply:
Hmm, I'd prefer the boring declarative way over some programmatic way, but I can see how inconsistencies can get frustrating. I don't know ;_; `/roll 2`
![image](https://user-images.githubusercontent.com/25330882/213808610-0210b708-f2f5-4d18-9693-36ddb083c43f.png)

Reply:
Declarative sorting works for me. If we want to hoist `vitest`, `msw`, memfs`, etc. to the top, we can add this line to the config:

```js
["^(vitest|msw|memfs)(/|$)"]
```

Reply:
OK, I'm for doing nothing